Library/Formula/m2c.rb

```diff
@@ -0,0 +1,44 @@
+class M2c < Formula
+  desc "Modula-2 to C compiler"
+  homepage "https://savannah.nongnu.org/projects/m2c/"
+  url "http://download.savannah.gnu.org/releases/m2c/0.7/m2c-0.7.tar.gz"
+  sha256 "b725ed617f376e1a321e059bf1985098e950965d5edab161c6b24526f10a59bc"
+
+  def install
+    # The config for "gcc" works for clang also.
+    cp "config/generic-gcc.h", "config/generic-clang.h"
+    system "./configure", "+cc=#{ENV.cc}"
+
+    # Makefile is buggy!
```

Comment:
I mentioned, all of these things are already fixed in master branch. There has been radical reorganization since the 0.7 release.